base/pkg/entry.jl

```diff
@@ -174,13 +174,13 @@ function status(io::IO, pkg::AbstractString, ver::VersionNumber, fix::Bool)
             isfile("METADATA",pkg,"url") || push!(attrs,"unregistered")
             LibGit2.isdirty(prepo) && push!(attrs,"dirty")
             isempty(attrs) || print(io, " (",join(attrs,", "),")")
-        catch
-            print(io, "broken-repo (unregistered)")
+        catch err
```

Comment:
Also, this is just a summary table of the currently installed packages, so throwing an error half way through seems weird.  The assumption that was unchecked here was that is was a git repo at all which should be fixed now.